answers-Misbah.js

```diff
@@ -0,0 +1,89 @@
+// Programming Basics: For Loops
+// These exercises are designed for practising "for" loops. Print all your results to the console.
+
+// Addition. Write a program to add up the numbers 1 to 20.
+let numList = 0
+
+for (let i = 1; i <= 20; i++){  
```

Comment:
we should add the numbers 1 to 20 to the numList think to use i

Reply:
what do you mean?!!  i think you understand the question as we should add 1 to 20 like this 1+20 !!! 
** the question says **ADD UP** the numbers 1 to 20..